### ISSUE TITLE
Is it possible to pass `errorBoundary` prop for single-spa-react at the point of calling singleSpa.registerApplication?

### ISSUE THREAD

**Issue by ahayes91** [single-spa/single-spa-react#119]
**Describe the bug or question**
We use singleSpa to register a number of singleSpaReact microfrontend apps. We would like to have a shared ErrorBoundary implementation across these apps (for consistent UX when apps fail for any reason). Ideally we would define that errorBoundary function once, and be able to supply it once at the point of registering each app.

However, it seems that `customProps` on singleSpa.registerApplication isn't powerful enough to supply errorBoundary to the react apps:

```javascript
export const { bootstrap, mount, unmount } = singleSpaReact({
  React,
  ReactDOM,
  rootComponent: App,
  // For example, `domElementGetter` can be handled by the container that registers each app
  errorBoundary, // but errorBoundary HAS to be supplied here
});
```

**To Reproduce**
For bugs, create a small git repo or codepen that reproduces the problem, and provide steps on how to see the issue. This is also helpful for questions.

I'll try and create one of these over the next couple of days to better showcase the question.

**Expected behavior**
I'd expect to be allowed to supply a common singleSpaReact prop in the customProps attribute of registerApplication:

```javascript 
function errorBoundary (err) {
  return <ErrorBoundaryMessage errorMessage={err.toString()} />; // UI component for consistency in our application
}

// loop through each app and register it:
    singleSpa.registerApplication({
      name: appName,
      customProps: {
        domElementGetter: getDomElementGetter(domElement),
        errorBoundary,
      },
    });
```

**Screenshots and/or console output**
It doesn't seem like the errorBoundary is supplied - if I force an error in one of my apps, it crashes. 

**Additional context**
Add any other context about the problem or question here.

**Comment by joeldenning** [comment]
I had some time tonight so I implemented this in #120

**Comment by ahayes91** [comment]
@joeldenning That is awesome, quick work, thanks so much - looks like exactly what we would need. I'll keep an eye on the PR and release notes 😁
